scheduler/cloudformation/template.go

```diff
@@ -305,15 +313,21 @@ func (t *EmpireTemplate) Build(app *scheduler.App) (interface{}, error) {
 			serviceProperties["Role"] = t.ServiceRole
 		}
 		resources[service] = map[string]interface{}{
-			"Type": "AWS::ECS::Service",
-			"Metadata": &serviceMetadata{
-				Name: p.Type,
-			},
+			"Type":       "AWS::ECS::Service",
 			"Properties": serviceProperties,
 		}
 
 	}
 
+	outputs[servicesOutput] = map[string]interface{}{
```

Comment:
Yeah, so far this is the only output. I've kinda opted to not add any extra outputs that don't actually get used programmatically by the backend, since I'm not sure what else we need (but I could totally see exposing the load balancer endpoints back to Empire so you can display that in `emp`).

Anyways, easy to add outputs moving forward.